src/aiovodafone/api.py

```diff
@@ -422,6 +452,16 @@ async def get_voice_data(self) -> dict[Any, Any]:
 
         return data
 
+    async def restart_router(self) -> None:
+        """Router restart."""
+        _LOGGER.debug("Restarting router %s", self.host)
+        payload = {
+            "restart": "Router,Wifi,VoIP,Dect,MoCA",
```

Comment:
If we restart `Router`, do we need to specify all the other sub-features ?

Reply:
That's what the exact same call as the UI is doing, so I assumed it's necessary. Haven't explicitly tried removing the other attrs though.

Reply:
Please add a comment so we remember for the future ;-)

Reply:
Feels a bit redundant, but okay: 961b21fe1599d48bbccb2260d975be7a5eeb0f40

Reply:
edit: commit is gone. I squashed :)